src/Nethermind/Nethermind.State.Test.Runner/Properties/launchSettings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "profiles": {
```

Comment:
Probably remove? On the other hand, it might be convenient if we add statetest1.json and we can always debug tests in this way.

Reply:
Thats why i added it

Reply:
ok but where is `statetest1.json`?

Reply:
it exists previously in the project.